File: 백준/Bronze/2480. 주사위 세개/주사위 세개.js
const fs = require("fs");
const filePath = process.platform === "linux" ? "/dev/stdin" : "./input.txt";
let input = fs.readFileSync(filePath).toString().split(" ");

solution(parseInt(input[0]), parseInt(input[1]), parseInt(input[2]));

function solution(A, B, C) {
  if (A == B && B == C) {
    console.log(10000 + A * 1000);
  } else if (A == B || B == C || C == A) {
    if (A == B || A == C) {
      console.log(1000 + A * 100);
    } else {
      console.log(1000 + B * 100);
    }
  } else {
    console.log(Math.max(A, B, C) * 100);
  }
}

File: 백준/Bronze/2480. 주사위 세개/주사위 세개.py
A,B,C = map(int,input().split())

if A == B and B == C :
  print(10000 + A * 1000)
elif A == B or B == C  or A == C :
  if A == B or B == C :
    print(1000 + B * 100)
  else :
    print(1000 + A * 100)
else :
  if A > B and A > C :
    print(100 * A)
  elif  A < B and B > C :
    print(100 * B)
  else :
    print(100 * C)